harness/determined/cli/user.py

```diff
@@ -51,13 +51,15 @@ def log_in_user(parsed_args: Namespace) -> None:
     token_store.set_active(username)
 
 
-@authentication.optional
 def log_out_user(parsed_args: Namespace) -> None:
-    auth = authentication.cli_auth
-    if auth is None:
+    det_obj = None
+    try:
+        det_obj = det_obj = Determined(master=parsed_args.master, user=parsed_args.user)
+    except (api.errors.UnauthenticatedException, api.errors.ForbiddenException):
         return
+    assert det_obj is None
     try:
-        client.logout()
+        det_obj.logout()
```

Comment:
is there still an issue with client.logout(), that we're just avoiding here?

Reply:
just avoiding it here. Because we want to explicitly pass the correct master in.